tools/monitoring/mysql/db_mysql.go

```diff
@@ -71,3 +75,42 @@ func (c DBConfig) dataStoreName(dbName string) string {
 
 	return fmt.Sprintf("%sunix(%s)/%s", cred, "/cloudsql/"+c.Instance, dbName)
 }
+
+// PubsubMsgHandler adds record(s) to ErrorLogs table in database,
+// after parsing build log and compares the result with config yaml
+func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumber int) error {
+	config, err := config.ParseYaml(configURL)
+	if err != nil {
+		return err
+	}
+
+	logs, err := logs.ParseLog(buildLogURL, config.CollectErrorPatterns())
+	if err != nil {
+		return err
+	}
+
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
+	stmt, err := tx.Prepare("INSERT INTO ErrorLogs VALUES()")
```

Comment:
Could you specify the column names in the insert statement? This makes insert statement still valid in case we want to add new columns in the future.

Reply:
if we specify column names, then a subtle change in column name would fail the statement. I think there is drawbacks either ways

Reply:
If we change the order of columns or add new ones, we will fail here, which is not desirable. If we are changing the column name, then the query should change IMO and you will be responsible to make sure that you handle cases for the old name.